production/helm/loki/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: loki
-description: Helm chart for Grafana Loki and Grafana Enterprise Logs supporting both simple, scalable and distributed modes.
+description: Helm chart for Grafana Loki and Grafana Enterprise Logs supporting simple, scalable and distributed modes.
```

Loki has three [deployment modes](https://grafana.com/docs/loki/latest/get-started/deployment-modes/)

```suggestion
description: Helm chart for Grafana Loki and Grafana Enterprise Logs supporting monolithic, simple scalable, and microservices modes.
```